src/main/java/com/zendesk/maxwell/schema/columndef/DateColumnDef.java

```diff
@@ -12,12 +11,10 @@ public DateColumnDef(String tableName, String name, String type, int pos) {
 	}
 
 	private static SimpleDateFormat dateFormatter;
-	private static final TimeZone tz = TimeZone.getTimeZone("UTC");
 
 	protected static SimpleDateFormat getDateFormatter() {
 		if ( dateFormatter == null ) {
 			dateFormatter = new SimpleDateFormat("yyyy-MM-dd");
-			dateFormatter.setTimeZone(tz);
```

Comment:
I remembered why I had done this: https://github.com/zendesk/open-replicator/issues/14 -- otherwise open-replicator will pick up maxwell's timezone, which is probably not what the user wants (they probably want the data interpreted "raw").